fimsauthz-api/fims-authz.yaml

```diff
@@ -65,15 +65,15 @@ paths:
                 - grant_type
                 - client_id
                 - scope
-                - timestamp
```

Comment:
@arkits I think its a good thing to have timestamp as a required field. Only change in the spec needed is to name the field as current_timestamp. This will ensure, folks don't send requests with old signatures.